R/assay5.R

```diff
@@ -1117,7 +1117,7 @@ LayerData.StdAssay <- function(
   layer_name <- layer[1L] %||% DefaultLayer(object = object)[1L]
   layer <- Layers(object = object, search = layer)[1L]
   # layer <- match.arg(arg = layer, choices = Layers(object = object))
-  if (is.na(layer)) {
+  if (is.null(layer)) {
```

Comment:
also had to change this to `is.null` because of recent to change to output

Reply:
it seems that NULL and NA are confusing at many cases. We may need to set a rule to distinguish when return NA and when return NULL